ext/ed.h

```diff
@@ -27,7 +27,7 @@ class SslBox_t; // forward reference
 #endif
 
 bool SetSocketNonblocking (SOCKET);
-
+bool SetFdCloexec (int);
```

Comment:
Can be `SOCKET`, there's a `typedef int SOCKET` in project.h (for consistency with `SetSocketNonblocking`).
Name it `SetSocketCloexec` for consistency as well?